<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MuLocalMusicTableAreaUI</class>
 <widget class="QWidget" name="MuLocalMusicTableAreaUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="localTableLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="MuPlayAddButton" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="localMatchBtn">
        <property name="text">
         <string>Match Music</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>214</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="MuSearchBox" name="localSearchLocalMusicBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="MuTableView" name="localMusicTableView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MuSearchBox</class>
   <extends>QLineEdit</extends>
   <header>MuSmallControls.h</header>
  </customwidget>
  <customwidget>
   <class>MuTableView</class>
   <extends>QTableView</extends>
   <header>MuTableview.h</header>
  </customwidget>
  <customwidget>
   <class>MuPlayAddButton</class>
   <extends>QWidget</extends>
   <header>MuSmallControls.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
